db/db_impl/db_impl.cc

```diff
@@ -423,16 +415,36 @@ Status DBImpl::ResumeImpl(DBRecoverContext context) {
     if (file_deletion_disabled) {
       // Always return ok
       s = EnableFileDeletions(/*force=*/true);
+      assert(s.ok());
```

Comment:
Nit:
we can turn this `assert(s.ok())` into a `assert(false)` following the line 423 in the `if (!s.ok())` branch.
I understand this (non-ok) should not happen now, but maybe it's better to "try" to log the error and then abort.

Reply:
Good point!

Reply:
Fixed